data/autoyast_sle15/pc_tools.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email/>
+        <reg_code>!!!!!!</reg_code>
```

Comment:
I guess you should inject ```<reg_code>{{SCC_REGCODE}}</reg_code>```

Reply:
yeah sure , for now I just running it locally so hardcode real code in file and this is just replacement for obvious reason to not leak it ...